helm-chart/README.md

```diff
@@ -0,0 +1,636 @@
+
+```bash
+helm upgrade my-formbricks formbricks/formbricks \
+  --set env.SMTP_HOST=new-smtp.example.com \
+  --set env.SMTP_PORT=587 \
+  --set env.NEW_CUSTOM_VAR=newvalue
+```
+
+This command updates the SMTP host and port, and adds a new custom environment variable.
+
+#### 2. Enabling or Disabling Features
+
+You can enable or disable features by updating their respective values:
+
+```bash
+# Disable Redis
+helm upgrade my-formbricks formbricks/formbricks --set redis.enabled=false
+
+# Enable Redis
+helm upgrade my-formbricks formbricks/formbricks --set redis.enabled=true
+```
+
+#### 3. Scaling Resources
+
+To adjust resource allocation:
+
+```bash
+helm upgrade my-formbricks formbricks/formbricks \
+  --set resources.limits.cpu=1 \
+  --set resources.limits.memory=2Gi \
+  --set resources.requests.cpu=500m \
+  --set resources.requests.memory=1Gi
+```
+
+#### 4. Updating Autoscaling Configuration
+
+To modify autoscaling settings:
+
+```bash
+helm upgrade my-formbricks formbricks/formbricks \
+  --set autoscaling.minReplicas=3 \
+  --set autoscaling.maxReplicas=10 \
+  --set autoscaling.metrics[0].resource.target.averageUtilization=75
+```
+
+#### 5. Changing Database Credentials
+
+To update PostgreSQL database credentials:
+To switch from the built-in PostgreSQL to an external database or update the external database credentials:
+```bash
+helm upgrade my-formbricks formbricks/formbricks \
+  --set postgresql.enabled=false \
+  --set postgresql.externalUrl="postgresql://newuser:newpassword@external-postgres-host:5432/newdatabase"
+```
+
+This command disables the built-in PostgreSQL and configures Formbricks to use an external PostgreSQL database. Make sure your external database is set up and accessible before making this change.
+
+### Using a Values File for Complex Upgrades
+
+For more complex upgrades or when you need to change multiple values, it's recommended to use a values file:
+
+1. Create a file named `upgrade-values.yaml` with your desired changes:
+
+   ```yaml
+   env:
+     SMTP_HOST: new-smtp.example.com
+     SMTP_PORT: "587"
+   resources:
+     limits:
+       cpu: 1
+       memory: 2Gi
+   autoscaling:
+     minReplicas: 3
+     maxReplicas: 10
+   traefik:
+     enabled: true
+   postgresql:
+     auth:
+       username: newuser
+       password: newpassword
+       database: newdatabase
+   ```
+
+2. Apply the upgrade using the values file:
+
+   ```bash
+   helm upgrade my-formbricks formbricks/formbricks -f upgrade-values.yaml
+   ```
+
+Remember to always backup your data before performing upgrades, especially when modifying database-related settings.
+
+## Support
+
+For support with the Formbricks Helm chart:
+- Open an issue on the [Formbricks GitHub repository](https://github.com/formbricks/formbricks)
+- Join our [community Discord](https://discord.com/invite/3YFcABF2Ts)
+- For enterprise support, contact us at support@formbricks.com
```

Comment:
change email address to `hola@formbricks.com`

Reply:
👍